Fixes issue #28346 - Table height attribute. (#28519)
mdn/content: files/en-us/web/html/element/table/index.md

  - : This attribute defines an alternative text that summarizes the content of the table. Use the {{htmlelement("caption")}} element instead.
- `width` {{Deprecated_inline}}
  - : This attribute defines the width of the table. Use the CSS {{cssxref("width")}} property instead.

>     NOTE - : `<table>` does not support the `height` attribute. Some browser have a non-standard interpretation of this attribute. For more info, visit [Setting height attribute](https://jsfiddle.net/zQNS4/1/)

Reviewer: ```suggestion

> **Note:** `<table>` does not support the `height` attribute. Some browsers have a non-standard interpretation of this attribute. For more info, visit [Setting height attribute](https://jsfiddle.net/zQNS4/1/)
```

And the JS fiddle does not provide "more info" because I expect it to be about browser compatibility.

>     NOTE - : `<table>` does not support the `height` attribute. Some browser have a non-standard interpretation of this attribute. For more info, visit [Setting height attribute](https://jsfiddle.net/zQNS4/1/)

Reviewer 2: Here's a suggestion:

```suggestion
 
> **NOTE:** While no HTML specification includes `height` as a `<table>` attribute, some browsers support a non-standard interpretation of height. The unitless value sets a minimum absolute height in pixels. If set as a percent value, the minimum table height will be relative to the height of the parent container.
```
I prefer not linking to the fiddle for a myriad of reasons, but mainly no example showing this will pass my "let's link to this" because we are showing using non-standard practices. On MDN we only want to show models of what people should be doing. Any example with `height` is what people shouldn't be doing.

Author: Got your point.

  - : This attribute defines an alternative text that summarizes the content of the table. Use the {{htmlelement("caption")}} element instead.
- `width` {{Deprecated_inline}}
  - : This attribute defines the width of the table. Use the CSS {{cssxref("width")}} property instead.

> - `height` {{Deprecated_inline}}

Reviewer 2: it's not really deprected, as it never officially existed. So, we shouldn't use the deprecated_inline macro. Instead, let's include the support for a never included in any spec  `height` attribute as a note rather than listing it in the list of attributes.

>   - : A table does not support height attribute.
>   - : Warning: Some browser misinterpret this attribute. For more info, visit [Setting height attribute](https://jsfiddle.net/zQNS4/1/)

Reviewer: ```suggestion
  - : `<table>` does not support the `height` attribute. Some browser have a non-standard interpretation of this attribute. For more info, visit [Setting height attribute](https://jsfiddle.net/zQNS4/1/)
```

>   - : Warning: Some browser misinterpret this attribute. For more info, visit [Setting height attribute](https://jsfiddle.net/zQNS4/1/)

Reviewer: Also, do not use jsfiddle in this case. You should be linking to a descriptive source (because there's no normative text for this) explaining which browsers and why.

- `width` {{Deprecated_inline}}
  - : This attribute defines the width of the table. Use the CSS {{cssxref("width")}} property instead.

> > **NOTE:** While no HTML specification includes `height` as a `<table>` attribute, some browsers support a non-standard interpretation of height. The unitless value sets a minimum absolute height in pixels. If set as a percent value, the minimum table height will be relative to the height of the parent container.

Reviewer 2: ```suggestion
> **NOTE:** While no HTML specification includes `height` as a `<table>` attribute, some browsers support a non-standard interpretation of height. The unitless value sets a minimum absolute height in pixels. If set as a percent value, the minimum table height will be relative to the height of the parent container.

```

- `width` {{Deprecated_inline}}
  - : This attribute defines the width of the table. Use the CSS {{cssxref("width")}} property instead.

> > **NOTE:** While no HTML specification includes `height` as a `<table>` attribute, some browsers support a non-standard interpretation of height. The unitless value sets a minimum absolute height in pixels. If set as a percent value, the minimum table height will be relative to the height of the parent container.

Reviewer: ```suggestion
> **Note:** While no HTML specification includes `height` as a `<table>` attribute, some browsers support a non-standard interpretation of `height`. The unitless value sets a minimum absolute height in pixels. If set as a percent value, the minimum table height will be relative to the height of the parent container.
```
